buddyboss/buddyboss-platform
# #998 - allowing reordering the default gender in the profile fields

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #998 .

### How to test the changes in this Pull Request:

1. Go to 'BuddyBoss > Profiles > Profile Fields'
2. Scroll down to 'Type' then select 'Gender'
3. See error. The Male and Female genders are set as default and they cannot be re-ordered.


http://prntscr.com/t46b4x

### Other information:
